docs/openapi.md

```diff
@@ -44,17 +44,19 @@ Check [swagger](https://swagger.io/) or [other tools](https://github.com/OAI/Ope
 ### Instructions
 
 1. Open https://editor.swagger.io/
-   - Alternatively, you can run your own instance of Swagger Editor
-     by running the following command in terminal:
+   - Alternatively, you can run your own instance of Swagger Editor by running the following command in terminal:
 
-     `docker run -p 8081:8080 swaggerapi/swagger-editor`
+    ```bash
+    docker run -p 8081:8080 swaggerapi/swagger-editor
+    ```
 
-     Then go to: http://localhost:8081/
-2. Load the OpenAPI description document (File > Import file)
-3. Check domain and port values for your Splunk instance and Authorize
-4. Select method-path pair (eg. GET - /splunk_ta_snow_settings/logging ) and "Try it out"
-5. Define parameters and "Execute"
-6. Copy curl value, paste to your terminal, ADD `-k` PARAMETER, and run
+    Then go to: http://localhost:8081/
+
+1. Load the OpenAPI description document (File > Import file)
+1. Check domain and port values for your Splunk instance and Authorize
+1. Select method-path pair (eg. GET - /splunk_ta_snow_settings/logging ) and "Try it out"
+1. Define parameters and "Execute"
+1. Copy curl value, paste to your terminal, ADD `-k` PARAMETER, and run
```

Comment:
incremental numbers were lost here

Reply:
Actually, the ordered list need to follow `\d+\.` convention. Hence, if in future we want to add any steps in between, we just have to write `1.` and not change the complete order of trailing steps/points.

Reply:
If we can automate this decision in https://github.com/splunk/addonfactory-ucc-generator/blob/develop/.markdownlint.yaml - that would be an ideal separate small PR.

Reply:
Raised PR #1414 for the same.